rockspec/apisix-master-0.rockspec

```diff
@@ -54,6 +54,7 @@ dependencies = {
     "skywalking-nginx-lua-plugin = 1.0-0",
     "base64 = 1.5-2",
     "dkjson = 2.5-2",
+    "lua-resty-redis-cluster = 1.1-0",
```

Comment:
https://luarocks.org/modules/steve0511/resty-redis-cluster

we can use this library now. @liuhengloveyou

Reply:
updated